admin/tabs/configs/differ/ConfigDiffer.js

```diff
@@ -47,7 +47,7 @@ export class ConfigDiffer extends Component {
                     placeholder: 'https://remote-host/',
                     field: 'remoteHost',
                     width: 200,
-                    options: XH.getConf('xhAppInstances')
+                    options: XH.getConf('xhAppInstances').filter(it => it != window.location.origin)
```

Comment:
This will not properly filter out the localhost:8080 in dev (since origin will == 'http://localhost:3000'), but I don't think development is where we are really concerned about this behavior